WORKSPACE

```diff
@@ -139,101 +143,32 @@ yarn_install(
-            "0.21.0",
-        ],
-    },
-    dockerfile = "//images/bazelbuild:Dockerfile",
-)
-
-container_dockerfile(
-    name = "alpine",
-    dockerfile = "//images/alpine:Dockerfile",
-)
-
-container_dockerfile(
-    name = "katacoda-lint",
-    dockerfile = "//images/katacoda-lint:Dockerfile",
-)
-
-container_dockerfile(
-    name = "golang-dind",
-    dockerfile = "//images/golang-dind:Dockerfile",
-)
-
-container_dockerfile(
-    name = "gcloud-in-go",
-    build_args = {
-        "GCLOUD_VERSION": "163.0.0",
-    },
-    dockerfile = "//legacy/images/gcloud-in-go:Dockerfile",
-)
-
-container_dockerfile(
-    name = "minikube-in-go",
-    build_args = {
-        "BAZEL_VERSION": "0.18.0",
-        "GCLOUD_VERSION": "163.0.0",
-    },
-    build_matrix = {
-        "KUBERNETES_VERSION": ["v1.9.6", "v1.8.10", "v1.7.15"],
-    },
-    dockerfile = "//legacy/images/minikube-in-go:Dockerfile",
-)
-
-container_dockerfile(
-    name = "tarmak-ruby",
-    build_args = {
-        "GCLOUD_VERSION": "206.0.0",
-        "GCLOUD_HASH": "d39293914b2e969bfe18dd19eb77ba96d283995f8cf1e5d7ba6ac712a3c9479a",
-    },
-    build_matrix = {
-        "RUBY_VERSION": ["2.4.4"],
-    },
-    dockerfile = "//legacy/images/tarmak/ruby:Dockerfile",
-)
-
-container_dockerfile(
-    name = "tarmak-sphinx-docs",
-    build_args = {
-        "GCLOUD_VERSION": "178.0.0",
-        "GCLOUD_HASH": "2e0bbbf81c11164bf892cf0b891751ba4e5172661eff907ad1f7fc0b6907e296",
-    },
-    dockerfile = "//legacy/images/tarmak/sphinx-docs:Dockerfile",
-)
-
-# Dependencies used for the cert-manager e2e image
-http_file(
-   name = "golang",
-   urls = ["https://dl.google.com/go/go1.11.linux-amd64.tar.gz"],
-   sha256 = "b3fcf280ff86558e0559e185b601c9eade0fd24c900b4c63cd14d1d38613e499",
-)
-
-http_file(
-    name = "dep_linux",
-    executable = 1,
-    urls = ["https://github.com/golang/dep/releases/download/v0.5.0/dep-linux-amd64"],
-    sha256 = "287b08291e14f1fae8ba44374b26a2b12eb941af3497ed0ca649253e21ba2f83"
-)
-
-http_archive(
-    name = "helm_linux",
-    sha256 = "0fa2ed4983b1e4a3f90f776d08b88b0c73fd83f305b5b634175cb15e61342ffe",
-    urls = ["https://storage.googleapis.com/kubernetes-helm/helm-v2.10.0-linux-amd64.tar.gz"],
-    build_file_content = """
-filegroup(
-    name = "helm",
-    srcs = [
-        "linux-amd64/helm",
-    ],
-    visibility = ["//visibility:public"],
-)
-""",
+# load("//def:container.bzl", "container_dockerfile")
+
+# container_dockerfile(
+#     name = "gcloud-in-go",
+#     build_args = {
+#         "GCLOUD_VERSION": "163.0.0",
+#     },
+#     dockerfile = "//legacy/images/gcloud-in-go:Dockerfile",
+# )
+
+# container_dockerfile(
+#     name = "minikube-in-go",
+#     build_args = {
+#         "BAZEL_VERSION": "0.18.0",
+#         "GCLOUD_VERSION": "163.0.0",
+#     },
+#     build_matrix = {
+#         "KUBERNETES_VERSION": ["v1.9.6", "v1.8.10", "v1.7.15"],
+#     },
+#     dockerfile = "//legacy/images/minikube-in-go:Dockerfile",
+# )
```

Comment:
Remove comments?

Reply:
I'd commented these out in case we want to re-enable the Navigator jobs again in future, but happy to remove altogether for now and we can `git revert` in future if needs be?

Reply:
I think it would be cleaner